Classes/Event/PostCheckFileReferenceEvent.php

```diff
@@ -39,13 +40,13 @@ class PostCheckFileReferenceEvent
      * This is the value of the currently looped uploaded file.
      * It contains one file out of $_FILES
      */
-    protected array $uploadedFile = [];
+    protected ?UploadedFile $uploadedFile;
```

Comment:
In constructor you expect a non-nullable UploadFile. So, I think we can remove the nullable here, right?

```suggestion
    protected UploadedFile $uploadedFile;
```